visitors

```diff
@@ -0,0 +1,33 @@
+import javax.servlet.http.HttpServletResponse;
+public class cookieclass extends HttpServlet {
+String name;
+public void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+PrintWriter out=res.getWriter();
+name=req.getParameter(&quot;txt&quot;);
+String cut=&quot;1&quot;;
+int chk=0;
+String nwcount=&quot;&quot;;
+Cookie cw = new Cookie(name,cut);
+cw.setMaxAge(24);
+res.addCookie(cw);
+Cookie [] StoreCookies = req.getCookies();
+int i,count;
+if(StoreCookies!= null)
+{
+for(i=0;i&lt;StoreCookies.length;i++)
+{
+String cname= StoreCookies[i].getName();
+String coccur = StoreCookies[i].getValue();
+if(name.equals(cname))
+{
+count = Integer.parseInt(coccur);
+count = count + 1;
+
+nwcount = Integer.toString(count);
+Cookie cc = new Cookie(name,nwcount);
+cc.setMaxAge(24);
+res.addCookie(cc);
+chk=1;
+}
+}
+}
```

Comment:
if(chk==1)
{
out.println(&quot;&lt;br&gt; Welcome,&lt;b&gt;&lt;i&gt;&lt;font size=&#39;+3&#39; color=&#39;red&#39;&gt; &quot; +name+ &quot; &lt;/font&gt;&lt;/i&gt;&lt;/b&gt;&lt;br&gt;

You are not a new visitor and you have been visiting for &quot; +nwcount+ &quot; times &quot;);
}
else
{
out.println(&quot;&lt;br&gt;Welcome,&lt;b&gt;&lt;i&gt;&lt;font size=&#39;+3&#39; color=&#39;red&#39;&gt; &quot; +name+ &quot; &lt;/font&gt;&lt;/i&gt;&lt;/b&gt;&lt;br&gt;

You are a new visitor and you have been registered&quot;);
}
}
}